src/cem_drift.F

```diff
@@ -468,12 +477,44 @@ subroutine cem_drift_rhs_bdf1
       call col3(w_h1n,cn,mu_n,npts) ! wk_h1n= mu_n * cN   (=h1)
       call col3(w_h1p,cp,mu_p,npts) ! wk_h1p= mu_p * cP  (=h1)
 
-      call axhelm3(wn,potent,w_h1n,h2,1,1)
-      call axhelm3(wp,potent,w_h1p,h2,1,1)
+      call axhelm3(wn,potent,w_h1n,w_h2,1,1)
+      call axhelm3(wp,potent,w_h1p,w_h2,1,1)
 
       call add2s2 (resN,wn, 1.0*dt,npts)
       call add2s2 (resP,wp,-1.0*dt,npts)
 
+!$ACC DATA PRESENT(cemface_nmn,cemface,cee,unxm,unym,unzm,mu_n,mu_p)
+!$ACC&     PRESENT(aream,cN,,cP)
+!$ACC PARALLEL LOOP
+      do i=1,ncemface_nmn(2) !FIXME, by Lan, it's not an efficient way
```

Comment:
A note that putting FIXME in the code is not an effective way to track issues. If code really needs to be merged but there are still problems then a Github issue should be created.

FIXMEs just get forgotten and then never fixed, like e.g. here:

https://github.com/NekCEM/NekCEM/blob/development/src/cem_drive.F#L779.